packages/cli/src/ui/components/messages/UserMessage.tsx

```diff
@@ -12,18 +12,25 @@ import { isSlashCommand as checkIsSlashCommand } from '../../utils/commandUtils.
 
 interface UserMessageProps {
   text: string;
+  width: number;
 }
 
-export const UserMessage: React.FC<UserMessageProps> = ({ text }) => {
+export const UserMessage: React.FC<UserMessageProps> = ({ text, width }) => {
   const prefix = '> ';
   const prefixWidth = prefix.length;
   const isSlashCommand = checkIsSlashCommand(text);
 
   const textColor = isSlashCommand ? theme.text.accent : theme.text.secondary;
 
   return (
-    <Box flexDirection="row" paddingY={0} marginY={1} alignSelf="flex-start">
-      <Box width={prefixWidth}>
+    <Box
+      flexDirection="row"
+      paddingY={0}
+      marginY={1}
+      alignSelf="flex-start"
+      width={width}
+    >
+      <Box width={prefixWidth} flexShrink={0}>
```

Comment:
It was a bug these prefixes didn't have flexShrink. It would cause the width of this component to shrink when the terminal width was small and the content after this wanted a larger width.